src/ColorVectorSpace.jl

```diff
@@ -354,4 +347,20 @@ typemax{T<:ColorTypes.AbstractGray}(::Type{T}) = T(typemax(eltype(T)))
 typemin{T<:ColorTypes.AbstractGray}(::T) = T(typemin(eltype(T)))
 typemax{T<:ColorTypes.AbstractGray}(::T) = T(typemax(eltype(T)))
 
+# deprecations
+function Base.one(::Type{C}) where {C<:Union{TransparentGray,AbstractRGB,TransparentRGB}}
+    Base.depwarn("one($C) will soon switch to returning 1; you might need to switch to `oneunit`", :one)
+    C(_onetuple(C)...)
+end
+_onetuple(::Type{C}) where {C<:Colorant{T,N}} where {T,N} = ntuple(d->1, Val(N))
+
+for f in (:min, :max)
+    @eval begin
+        @deprecate($f{T<:Gray}(x::Number, y::AbstractArray{T}),
+                   @compat $f.(x, y))
```

Comment:
Since the package no longer supports 0.4, you can drop the `@compat` for dot broadcast.